### ISSUE TITLE
Provide the pretraining trainer code with generative attention masking

### ISSUE THREAD

**Issue by Sustella** [bowang-lab/scGPT#21]
This is amazing work that extends GPT to genomic data for the first time.

If my data is not blood or bone marrow cells, I need to use your GPT pretraining workflow to train on my dataset.

When are you going to provide the pretraining code with generative attention masking?

Thank you very much!

**Comment by subercui** [comment]
Thank you for your interest in our work. We expect to release the pretraining code this weekend.

In the meantime, I think there is a better solution for your need. We just released several new checkpoints [here](https://github.com/bowang-lab/scGPT#pretrained-scgpt-checkpoints). The pretrained `whole-human` model can likely be an option for you.

**Comment by Sustella** [comment]
Thank you very much!

**Comment by Mczn** [comment]
Hey, I'd love to use your pretraining workflow to train scGPT on a mice dataset. Is there an update on when the pretraining code is going to be released? I'd be very curious to try it out.

Thanks for the great work

**Comment by subercui** [comment]
@Mczn Thank you for your interest! We are wrapping it up and plan to release the pretraining code by this weekend.
